content/docs/cml-with-dvc.md

```diff
@@ -75,6 +75,25 @@ few examples for some of the most frequently used providers:
 - `AWS_SECRET_ACCESS_KEY`
 - `AWS_SESSION_TOKEN` **(optional)**
 
+<toggle>
```

Comment:
As usual, I've placed a couple nested tabs inside `/doc/cml-with-dvc` for testing. This pull request will be a draft until its approved and I delete this code.